Add contributing file (#9)
ExpediaGroup/helm-charts: CONTRIBUTING.md

# Contributing new Helm Charts

> If you would like to contribute with a new Expedia Group helm chart, fork this GitHub repository, create a branch, and send a Pull Request.

Reviewer: ```suggestion
If you would like to contribute a new Expedia Group Helm chart:
 * fork this GitHub repository and create a branch
 * create a a folder for your chart under `charts` and add the relevant YAML, template and related files (refer to an existing chart for reference)
* submit a Pull Request with these changes
```

## Release process
Unfortunately this process is not automated yet, so please bear with us 🦄

> 1. Start by adding the new chart with the sources to the `charts` directory in the `master` branch.

Reviewer: Ah OK, so maybe I'm misunderstanding this, do they need to do this as part of the above PR or does someone else need to do the above in master later?

Author: I would expect that until we put some automation in place, that the contributor to open two pull requests.
One for master with the source of the charts, and one for the gh-pages with the updated index and the chart binaries.

Let me change the text because yeah, this is not clear.

2. Next, package your new chart (you can use `helm package .`) and add it to the `gh-pages` branch.

> 3. Finally, you will need to update the index file (you can use `helm repo index .`)

Reviewer: Does this then need to be committed too?

Author: updated the text

# Contributing new Helm Charts

Note: Unfortunately the process to publish new Helm Charts is not automated yet, so please bear with us while we explain this two step process 🦄

If you would like to contribute a new Expedia Group Helm chart you will need to open 2 pull requests.

## Publish Sources

* Start by forking this Github repository and creating a new branch from master.
* If this is a new application, create a new folder under `charts`.
* Add the relevant Yaml files (refer to an existing chart for reference).
* Submit a Pull Request with these changes.

## Publish Helm Charts

> * Switch to the `gh-pages` and create a new branch in your fork.

Reviewer: ```suggestion
* Switch to the `gh-pages` branch and create a new branch from this in your fork.
```

> * Package your chart into a `.tgz` file. See [helm package](https://helm.sh/docs/helm/helm_package/).

Reviewer: ```suggestion
* Package your chart into a `.tgz` file. See [Helm Package](https://helm.sh/docs/helm/helm_package/).
```

* Add your file under the `repo` folder.

> * Finally, you will need to update the index file. See [helm repo index](https://helm.sh/docs/helm/helm_repo_index/). 

Reviewer: ```suggestion
* Finally, you will need to update the index file. See [Helm Repo Index](https://helm.sh/docs/helm/helm_repo_index/). 
```
